woo-razorpay.php

```diff
@@ -160,6 +161,7 @@ public function __construct($hooks = true)
             $this->icon =  "https://cdn.razorpay.com/static/assets/logo/payment.svg";
             // 1cc flags should be enabled only if merchant has access to 1cc feature
             $is1ccAvailable = false;
+            $isAccCreationAvailable = true;
```

Comment:
@poojarazor There is no, SRF feature flag for this?

Reply:
we have SRF for this "one_cc_store_account"